backend/src/main/java/com/redhat/cloud/notifications/recipients/itservice/pojo/AccountRelationship.java

```diff
@@ -0,0 +1,41 @@
+package com.redhat.cloud.notifications.recipients.itservice.pojo;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "allOf",
+        "by"
+})
+public class AccountRelationship {
+
+    @JsonProperty("allOf")
+    private List<String> allOf = List.of("primary_email");
+
+    @JsonProperty("by")
+    private By__1 by;
```

Comment:
Was this name auto generated?

Reply:
Yes

Reply:
Could you add some steps on how to generate it again in case we need to sync?

Reply:
sure. where do you want it to be? in the readme of the application?

Reply:
a new section on the README would be nice

Reply:
done :)